src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/Program.cs

```diff
@@ -181,29 +182,18 @@ private static void ShowHeader()
         private class HelpWithHeader : HelpBuilder
         {
             public HelpWithHeader(IConsole console)
-                : base(console, maxWidth: 90)
+                : base(console, maxWidth: ConsoleUtils.Width)
             {
             }
 
-            public override void Write(ICommand command)
+            protected override void AddSynopsis(ICommand command)
             {
                 ShowHeader();
 
-                WriteString("Makes a best-effort attempt to upgrade .NET Framework projects to .NET 5.");
-                WriteString("This tool does not completely automate the upgrade process and it is expected that projects will have build errors after the tool runs. Manual changes will be required to complete the upgrade to .NET 5.");
-                WriteString("This tool's purpose is to automate some of the 'routine' upgrade tasks such as changing project file formats and updating APIs with near-equivalents in .NET Core. Analyzers added to the project will highlight the remaining changes needed after the tool runs.");
-
-                base.Write(command);
-            }
-
-            private void WriteString(string input)
-            {
-                foreach (var line in SplitText(input, MaxWidth))
-                {
-                    Console.Out.WriteLine(line);
-                }
-
-                Console.Out.WriteLine();
+                const string Title = "Makes a best-effort attempt to upgrade .NET Framework projects to .NET 5.\n\n" +
```

Comment:
nit: use {Environment.Newline}

Reply:
I'm using a const string, so that doesn't work. This should be moved out to a resource file anyway, so I'll leave it as-is and create a task to use resources.